### ISSUE TITLE
Kiwix uses the end-of-life org.kde.Platform//5.15 runtime

### ISSUE THREAD

**Issue by LaurentBonnaud** [flathub/org.kiwix.desktop#8]
Hi,

here is the problem:
```
$ flatpak update
Looking for updates…
Info: org.kde.Platform//5.15 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Applications using this runtime:
   org.kiwix.desktop
```

**Comment by mgautierfr** [comment]
Fixed in kiwix/kiwix-build#526
Please for next release of kiwix-desktop to see the fix.

**Comment by LaurentBonnaud** [comment]
Thanks a lot for the fix!
